### DISCUSSION TITLE
Check $exists on a field in a document in v3,v4 with typescript and mongodb

### DISCUSSION THREAD

**Discussion by chrisschaub** [prisma/prisma#15007] [Q&A]
I am using prisma 3 or 4, trying to run this query with typescript:
```
	const results = await prisma.name.findMany({
		where: {
			nameParts: { $exists: false }
		}
	});
```
but keep getting this  error:
```
Object literal may only specify known properties, and '$exists' does not exist in type 'StringNullableListFilter'.
11    nameParts: { $exists: false }
```

I am trying to find a way to select document where this field does not exist, like you would in a straight mongodb query. I've been hunting around and only found a solution for prisma v1, but not v3,4. I think this is different from this issue https://github.com/prisma/prisma/issues/5022 .

**Comment by austincrim** [comment]
Hey @chrisschaub 👋,

If you want to use plain MongoDB operators like `$exists`, you need to use the [`findRaw` method](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#findraw).

```ts
const results = await prisma.name.findRaw({
  filter: {
    nameParts: { $exists: false }
  }
});
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
